docusaurus/docs/getting-started.md

```diff
@@ -3,8 +3,18 @@ sidebar_position: 1
 title: Getting Started
 ---
 
+import LiteYouTubeEmbed from 'react-lite-youtube-embed';
+import 'react-lite-youtube-embed/dist/LiteYouTubeEmbed.css'
+
 # Getting Started
 
+0. Checkout our tutorial on how to use Janush!
+
+ <LiteYouTubeEmbed 
```

Comment:
Can we format it a bit nicer?

Reply:
@przemyslaw-rzasa-cp  
you mean a better text description or better visual formatting of the embedded youtube video?

Reply:
Embedded formatting only